nodejs/eks/cni/aws-k8s-cni.yaml

```diff
@@ -104,10 +104,6 @@ spec:
               valueFrom:
                 fieldRef:
                   fieldPath: spec.nodeName
-            - name: WATCH_NAMESPACE
-              valueFrom:
-                fieldRef:
-                  fieldPath: metadata.namespace
```

Comment:
Did this go away due to a change in the `1.5.3` upstream?

Reply:
Yes. I noted this in the commit:

> Based on: https://git.io/fj56i

Here's the upstream [PR](https://github.com/aws/amazon-vpc-cni-k8s/pull/578).